bundles/org.openhab.core.io.transport.serial/src/main/java/org/openhab/core/io/transport/serial/SerialPortManager.java

```diff
@@ -41,9 +41,12 @@
     }
 
     /**
-     * Gets the serial port identifiers.
+     * Gets the discovered serial port identifiers.
      *
-     * @return stream of serial port identifiers
+     * {@link SerialPortProvider}s may not be able to discover any or all identifiers.
+     * When the port name is known, the preferred way to get an identifier is by using {@link #getIdentifier(String).
```

Comment:
A missing curly bracket? Isn't it?

```suggestion
     * When the port name is known, the preferred way to get an identifier is by using {@link #getIdentifier(String)}.
```

Reply:
Well spotted! The curly has returned from its abscence. :-)